xbmc/guilib/GUIFontTTF.cpp

```diff
@@ -732,7 +732,7 @@ bool CGUIFontTTFBase::CacheCharacter(wchar_t letter, uint32_t style, Character *
       m_posX += -bitGlyph->left;
 
     // check we have enough room for the character
-    if ((m_posX + bitGlyph->left + bitmap.width) > static_cast<int>(m_textureWidth))
+    if ((m_posX + bitGlyph->left + bitmap.width) > m_textureWidth)
```

Comment:
right. i guess we need to explicitly cast both sides then to have it work everywhere.